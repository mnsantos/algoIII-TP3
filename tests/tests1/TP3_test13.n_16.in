16 46
1 10
7 15
2 10
3 10
3 6
2 14
15 16
13 14
12 15
12 14
9 16
9 14
8 15
5 15
1 8
1 2
1 11
1 15
10 11
3 14
7 11
4 5
4 12
6 15
1 9
1 6
1 12
1 7
1 14
1 4
1 13
3 13
10 12
1 3
6 12
9 10
8 9
9 12
5 8
5 12
1 5
7 9
5 16
4 16
1 16
2 16
0
