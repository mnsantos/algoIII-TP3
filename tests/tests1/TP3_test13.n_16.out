24 4 1 5 12 15
